src/sensors/sensors_tests/HC-SR04/HC-SR04_simple_test.py

```diff
@@ -0,0 +1,61 @@
+# Code based (a lot) on code from https://github.com/makersdigest/T07-HC-SR04-Ultrasonic-Sensor
+# Imports python modules
+import RPi.GPIO as GPIO
+import time
+import sys
+import argparse
+
+
+def run_hc_sr04(trigger_pin, echo_pin):
+    try:
+        while True:
+			# "Warm up" trigger pin...
+            GPIO.output(trigger_pin, GPIO.LOW)           # Set pin to LOW
+            time.sleep(2)                                # Wait 2 seconds
+
+            GPIO.output(trigger_pin, GPIO.HIGH)          # Set pin to HIGH
+            time.sleep(0.0001)                           # wait 10 microseconds (10us HIGH pulse)
```

Comment:
10 microseconds is 0.00001s. I believe sensor docs said the pulse to be **at least 10us**, so should work anyways. But we could change the value to match the actual pulse width.
```suggestion
            time.sleep(0.00001)                           # wait 10 microseconds (10us HIGH pulse)
```

Reply:
Thanks for your remark!
To answer your question: it is only a script for testing how the sensor is working or how to work with the sensor. I don't think it will be a part of the final solution for the sensor module.